src/openzaak/components/documenten/api/viewsets.py

```diff
@@ -1052,3 +1057,29 @@ def create(self, request, *args, **kwargs):
 
         output_serializer = self.serializer_class(instances, many=True)
         return Response(output_serializer.data, status=status.HTTP_201_CREATED)
+
+
+class DocumentRegistrerenViewSet(viewsets.ViewSet):
```

Comment:
Could you use drf-spectacular's `extend_schema` to add a more readable name and description? Similar to other endpoints

![afbeelding](https://github.com/user-attachments/assets/dff0ee7c-4deb-4300-8343-74a0a5d4bdd7)